CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ Changes
 
 22.01 to 22.05
 --------------
+* [Developer]: Add `--no-yarn` to skip the yarn build in `run.sh`
```

Comment:
Need to update this to mention `--prod` option.

Reply:
Updated in 60965d0